### ISSUE TITLE
ValueError: With local_files_only set to None, you must first locally save the tokenizer in the following path: 'laion/CLIP-ViT-bigG-14-laion2B-39B-b160k' with `pad_token` set to '!'.

### ISSUE THREAD

**Issue by oftenliu** [TencentARC/PhotoMaker#71]
error happen,like this:
ValueError: With local_files_only set to None, you must first locally save the tokenizer in the following path: 'laion/CLIP-ViT-bigG-14-laion2B-39B-b160k' with `pad_token` set to '!'.

**Comment by XinPeiHou** [comment]
I also encountered the same problem. Have you solved it?

**Comment by oftenliu** [comment]
no,i am download the 'laion/CLIP-ViT-bigG-14-laion2B-39B-b160k' from huggingface now, good luck for me!

**Comment by ppppppz** [comment]
> no,i am download the 'laion/CLIP-ViT-bigG-14-laion2B-39B-b160k' from huggingface now, good luck for me!

Hello, what models are needed extra? I downloaded the 'laion/CLIP...' and 'openai/CLIP...'. Thx for answer!  :)
